improver_tests/acceptance/test_generate_realizations.py

```diff
@@ -115,9 +113,19 @@ def test_realizations(tmp_path):
     acc.compare(output_path, input_path)
 
 
+def test_error_no_realizations_count(tmp_path):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function name "test_error_no_realizations_count" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5435102)